gptel-context.el

```diff
@@ -210,6 +252,12 @@ If selection is active, removes all contexts within selection."
     (when-let ((ctx (gptel-context--at-point)))
       (delete-overlay ctx)))))
 
+(defun gptel-context--remove-file (path)
+  "Remove the file at PATH from the gptel context."
+  (setf (alist-get path gptel-context--alist nil 'remove #'equal)
+	nil)
+  (message "File \"%s\" removed from context." path))
```

Comment:
This is fine when removing individual files.  When removing a directory however, this will emit one message per removed file, which can be a lot.  I think this messaging needs to happen elsewhere.

Reply:
Addressed here: https://github.com/karthink/gptel/pull/438/commits/f73584eebfd6f483cd98e02e89f08b0e01f336b4

Reply:
Perfect, I think we can merge this now, unless you plan to make any more changes?

Reply:
Great. Yes, let’s merge it.